catalog/landing-zone/namespaces/logging.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
   name: logging
   annotations:
     cnrm.cloud.google.com/organization-id: "123456789012" # kpt-set: ${org-id}
+    cnrm.cloud.google.com/ignore-clusterless: "true"
```

Comment:
It does, but there's a theoretical universe where it might try to do something with them eventually. So I figured it's safest to ignore all these resources.